style.css

```diff
@@ -266,22 +351,31 @@ button[type="submit"] {
     height: 3.5rem;
     padding: 0;
   }
+
   #account {
     display: block;
   }
+
   main {
     max-width: 20rem;
   }
+
   h1 {
     font-size: 1.9rem;
     margin-top: 1.5rem;
   }
+
   h2 {
     font-size: 1.625rem;
     margin-top: -0.9rem;
     margin-bottom: 1.7rem;
   }
+
   #existing-account {
     margin: 0;
   }
+
```

Comment:
I think `#not-yours` is also `display: none`... you could add both to the list at the top of all the things we don't want to display... T